bigip/resource_bigip_ltm_virtual_server.go

```diff
@@ -45,6 +45,13 @@ func resourceBigipLtmVirtualServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"state": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "enabled",
+				ValidateFunc: validateEnabledDisabled,
+				Description:  "Specifies whether the virtual server and its resources are available for load balancing. The default is Enabled",
```

Comment:
Since the validator is strict about the values supported for this attribute, we should specify the exact values supported here, i.e. `enabled` and `disabled.

Reply:
ValidateFunc throws error message if user provided attribute value other than `enabled` or `disabled`